Shopify/dawn
# Language currency picker announcement bar

### PR Summary: 
Added language/currency picker into the announcement bar.

### Why are these changes introduced?

Fixes #.

### What approach did you take?
**
Added a Currency selector and Language selector in the schema
Added some classes and styling to the language & currency selector in announcement bar
add language & currency selector in announcement-bar.liqud
changed header drawer to pull currency and language information from global localization object vs. selector so that even if currency and language selector are unchecked in the header, currency and/or language selectors will be present in mobile drawer and absent from announcement bar when in mobile view.

### Other considerations
N/A


### Visual impact on existing themes
Merchants can customize where their language/currency selectors go. They can place them in the announcement bar or header or both. 


### Testing steps/scenarios

Announcement bar
- [ ] Deselect Enable country/region selector and Enable language selector
- [ ] View Announcement bar in mobile, tablet and desktop with selected and unselected currency/language selectors

Header
- [ ] Deselect Enable country/region selector and/or Enable language selector

Mobile Testing
1.  Deselect Enable country/region selector and/or Enable language selector
 from header
2. Keep Enable country/region selector and/or Enable language selector in announcement bar
3. Test in mobile settings and ensure language and currency selector appear in mobile drawer

Additional testing:
- [ ] Test color scheme
- [ ] Remove languages and leave only one to test for this conditional `localization.available_languages.size > 1`
- [ ] Remove countries and leave only one to test for this conditional `localization.available_countries.size > 1`

### Demo links
- [Editor](https://os2-demo.myshopify.com/admin/themes/140176261142/editor)

### Checklist
